README.md

```diff
@@ -1 +1,180 @@
-# Module2-Capstone
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+ 
+
+  <h3><b>Module 2 capstone</b></h3>
```

Comment:
- Optional
I would strongly recommend choosing a proper name for this project. Since you will be adding this project to your portfolio, any recruiter visiting your project repo will be noticing how your README looks. Remember it is super important to have a properly well-documented README file.